### PR TITLE
Added embrace-config-schema.json for tracking and documenting local configs

### PR DESCRIPTION
**Pull request by priettt**

### CODE REVIEW

File: embrace-config-schema.json [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+      "description": "Enables NDK crash capture. Defaults to false.",
+      "type": "boolean"
+    },
+    "sdk_config": {
+      "type": "object",
+      "properties": {
+        "automatic_data_capture": {
+          "type": "object",
+          "properties": {
+            "memory_info": {
```
Comment (bidetofevil) [inline embrace-config-schema.json:28]: Is basically doesn't work and is deprecated (or will be). Should we not include this here so it's not suggested?

File: embrace-config-schema.json [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+              "type": "boolean"
+            }
+          },
+          "minProperties": 1
+        },
+        "app": {
+          "type": "object",
+          "properties": {
+            "report_disk_usage": {
+              "description": "The SDK collects the disk usage for the app. You can disable this if your app has a large number of local files to prevent excessive resource usage, especially on lower-end devices. Defaults to true.",
```
Comment (bidetofevil) [inline embrace-config-schema.json:183]: I don't understand this description...

**Review by bidetofevil — APPROVED** [review]
LGTM. I have lots of comments about the individual descriptions but I'll just do a PR to change them later :)

**Comment by priettt** [comment]
> LGTM. I have lots of comments about the individual descriptions but I'll just do a PR to change them later :)

@bidetofevil I'm using the descriptions that were already in the schema uploaded to schemastore, but we definitely need to improve them so go ahead!

File: embrace-config-schema.json (hunk 1 above)
Comment (priettt) [reply embrace-config-schema.json:29]: can I remove it from the swazzler and SDK code?

**Pull request merged**